tables/lg-ug-g1.utb

```diff
@@ -0,0 +1,58 @@
+# liblouis: Luganda Grade 1 Braille
+#
+#-display-name: Luganda Grade 1
+#-index-name: Luganda Grade 1
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#
+#+language: lg
+#+region: ug
+#+type: literary
+#+contraction: no
+#+grade: 1
+#+dots: 6
+#+direction: both
+#
```

Comment:
All the South-African have a "system" metadata tag that begins with "ubc-" ("Unified Braille Code"). I wonder if the same applies for this table as it is clearly derived from UEB as well.

Reply:
I don't know if this applies in Uganda. I will try to find out.

Reply:
Thanks. Maybe Christo de Klerk knows.

Reply:
I have not been able to find anything out about this.

Reply:
OK, that's fine. Thanks for checking.